API.md

```diff
@@ -3432,7 +3432,8 @@ Default value: `'error'` (return a Bad Request (400) error response).
 
 A [`failAction` value](#lifecycle-failAction) which determines how to handle failed validations.
 When set to a function, the `err` argument includes the type of validation error under
-`err.output.payload.validation.source`.
+`err.output.payload.validation.source`. The default error that would otherwise have been logged
+ or returned can be accessed under `err.data.defaultError`.
```

Comment:
Just noticed, is it me or do we have a line return in the middle of the phrase?

Reply:
Nevermind I checked the file rendering on GitHub and it displays properly. I might have been confused by the diff tool I guess.